### ISSUE TITLE
RSS not working with valid RSS feed

### ISSUE THREAD

**Issue by cmgorton** [forem/forem#12087]
**Describe the bug**

Valid RSS feeds are not working for several users. 

One user is using Gatsby with MDX and Styled Components. 
This is the error they saw when trying to set up RSS
![RSS error)](https://user-images.githubusercontent.com/21039864/103420684-2550e680-4b5e-11eb-87d9-2fdeb95e51e5.png)

The other user is trying to update their Ghost blog RSS feed:
_"Clicking the "Fetch feed now" button does not create any draft articles in my dashboard"_

**Expected behavior**
Both users have valid RSS feeds. They should be able to add and update their feeds.

**Screenshots**

**Additional context**

I will ask both users to add more context here if they are able too.

**Comment by palebp** [comment]
Clicking "Fetch feed now" button -> Getting the message "Your profile was successfully updated." -> No drafts in Dashboard

**Comment by zorfling** [comment]
Similar to @palebp, I was able to click fetch feed now and have it appear to work, but not create any new drafts. (It had previously worked - however I'd recently updated to use MDX)

Then I changed the feed URL to my Medium feed, which did work and added new drafts. 

Then when I tried to change it back to my Gatsby feed I get the screenshotted error above - "Feed url is not a valid RSS/Atom feed". I tested my feed with a number of RSS validators which all suggested it was valid.

Feed in question is https://chriscolborne.com/rss.xml

Thanks!

**Comment by rhymes** [comment]
I'm going to take a look at this. Thanks @cmgorton and @zorfling for the report

**Comment by rhymes** [comment]
Hi, sorry for the delay @zorfling, I checked https://chriscolborne.com/rss.xml with [W3C feed validation service](https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fchriscolborne.com%2Frss.xml) says it's valid.

TLDR; can you try again, I was able to import them locally and I haven't found anything wrong logged in the errors.

I also checked manually with the code that's running on dev.to and it says it's valid:

```ruby
Feeds::ValidateUrl.call("https://chriscolborne.com/rss.xml")
=> true
```

I checked again by actually parsing the code with our parser and the data is there:

```ruby
[11] pry(main)> xml = HTTParty.get("https://chriscolborne.com/rss.xml", timeout: 10).body
[12] pry(main)> Feedjira.parse(xml)
=> #<Feedjira::Parser::RSS:0x00007fa980722500
 @description="An Aussie software engineer, talking tech and more.",
 @entries=
  [#<Feedjira::Parser::RSSEntry:0x00007fa98071a058
    @content=
     "<p>Running some code after changing state is a common desire. Let’s look at how to do that with hooks.
...
[14] pry(main)> Feedjira.parse(xml).entries.length
=> 16
```

and `curl` confirms it:

```shell
> curl https://chriscolborne.com/rss.xml | rg "<item>" | wc -l
      16
```

there are indeed 16 items and Ruby sees them. I then proceeded to try to reproduce the behavior locally by doing the following:

1. adding https://chriscolborne.com/rss.xml to the `/settings/extensions` section for RSS

![Screenshot_2021-01-04 Settings - DEV(local) Community 🌱](https://user-images.githubusercontent.com/146201/103538942-70893300-4e97-11eb-8633-31978344d170.png)

2. saved by pressing "Save feed settings" (I also tried with fetch feed now)
3. waited a while and found these in the dashboard


![Screenshot_2021-01-04 Dashboard - DEV(local) Community 🌱](https://user-images.githubusercontent.com/146201/103538968-7bdc5e80-4e97-11eb-9066-6fea5e663022.png)

**Comment by rhymes** [comment]
@palebp what is the URL of your feed? Thank you!

**Comment by palebp** [comment]
Hey, thanks for looking into it. My RSS feed: https://www.rockandnull.com/rss/

**Comment by zorfling** [comment]
Thanks @rhymes for looking! Glad to see I'm not completely crazy (the feed is valid).

Unfortunately I'm still getting the feed is not valid error when I try to change it back. Could it be some sort of caching issue with my particular account?

**Comment by rhymes** [comment]
@palebp same outcome as with @zorfling's feed. I'm going to investigate further and report back, this is odd indeed

**Comment by rhymes** [comment]
Hi @zorfling I can see your articles imported, can you verify it's working? You should have them all in the Dashboard.

Hi @palebp, I wasn't able to find your DEV username to verify the same, can you tell us what's your DEV account?

**Comment by zorfling** [comment]
Hi @rhymes - yes I've just tested again now (including changing it to medium and back which is what showed the issue in the first place) and it seems to be working.

Thanks!

**Comment by palebp** [comment]
My DEV username is: dev.to/rockandnull/

I think is working again for newer articles. It does not import older articles though. Is this expected?

**Comment by maestromac** [comment]
@palebp that's because the older feed was previously imported by a different user. Looks like one of the authors of some of those feed. Please reach out to yo@dev.to and we'll help you resolve this.

**Comment by cmgorton** [comment]
Thanks for letting us know @palebp . And thanks for the help her Rhymes and Mac. I am going to go a head and close this issue. Please reach out to yo@dev.to like Mac mentioned and we can help resolve the rest of the issue!
